File: app/src/main/java/com/kelsos/mbrc/domain/TrackInfo.java
package com.kelsos.mbrc.domain;

public class TrackInfo {
  public String artist;
  public String title;
  public String album;
  public String year;

  public TrackInfo(String artist, String title, String album, String year) {
    this.artist = artist;
    this.title = title;
    this.album = album;
    this.year = year;
  }

  public String getArtist() {
    return artist;
  }

  public String getTitle() {
    return title;
  }

  public String getAlbum() {
    return album;
  }

  public String getYear() {
    return year;
  }
}

File: app/src/main/kotlin/com/kelsos/mbrc/domain/TrackInfo.kt
package com.kelsos.mbrc.domain

data class TrackInfo(var album: String, var artist: String, var path: String, var title: String, var year: String) {

  constructor() : this("", "", "", "", "")

  fun isEmpty(): Boolean {
    return album.isNullOrEmpty()
        && artist.isNullOrEmpty()
        && path.isNullOrEmpty()
        && title.isNullOrEmpty()
        && year.isNullOrEmpty()
  }
}
